content-services/content-sources-backend
# Fixes 3278: migrate tasking to pgx

## Summary

migrates our tasking system to pgxv5

## Testing steps

1) test the app normally, triggering a snapshot and watch it complete without error
2) Turn on pgx logging and set log level to trace:

```
logging:
  level: trace
  console: true

tasking:
  pgx_logging: true
  heartbeat: 1m
  worker_count: 3
```

verify that you see tasking logs

## Checklist
